src/main/java/com/rackspace/ceres/app/services/DataTablesStatements.java

```diff
@@ -101,42 +98,48 @@ private void buildDownsampleStatements(DownsampleProperties downsampleProperties
 
     downsampleProperties.getGranularities()
         .forEach(granularity -> {
-
-          downsampleInserts.put(granularity.getWidth(),
-              "INSERT INTO " + tableNameDownsampled(granularity.getWidth(),
-                  granularity.getPartitionWidth())
-                  + " ("
-                  + String.join(",",
-                  TENANT, TIME_PARTITION_SLOT, SERIES_SET_HASH, AGGREGATOR, TIMESTAMP, VALUE)
-                  + ")"
-                  + " VALUES (?, ?, ?, ?, ?, ?)"
-          );
-
-          downsampleQueries.put(granularity.getWidth(),
-              "SELECT " + String.join(",", TIMESTAMP, VALUE)
-                  + " FROM " + tableNameDownsampled(granularity.getWidth(),
-                  granularity.getPartitionWidth())
-                  + " WHERE " + TENANT + " = ?"
-                  + "  AND " + TIME_PARTITION_SLOT + " = ?"
-                  + "  AND " + SERIES_SET_HASH + " = ?"
-                  + "  AND " + AGGREGATOR + " = ?"
-                  + "  AND " + TIMESTAMP + " >= ? AND " + TIMESTAMP + " < ?"
-          );
-
-          downsampleDeletes.put(granularity.getWidth(),
-              "DELETE FROM " + tableNameDownsampled(granularity.getWidth(),
-                  granularity.getPartitionWidth())
-                  + " WHERE " + TENANT + " = ?"
-                  + "  AND " + TIME_PARTITION_SLOT + " = ?"
-          );
-
-          downsampleDeletesWithSeriesSetHash.put(granularity.getWidth(),
-              "DELETE FROM " + tableNameDownsampled(granularity.getWidth(),
-                  granularity.getPartitionWidth())
-                  + " WHERE " + TENANT + " = ?"
-                  + "  AND " + TIME_PARTITION_SLOT + " = ?"
-                  + " AND series_set_hash = ?"
-          );
+          try {
```

Comment:
I dont see any reason to have so many different try/catch blocks since the catch is the same for each one